slackcli/utils.py

```diff
@@ -62,6 +62,13 @@ def print_messages(source_name, count=20):
     for message in messages[::-1]:
         print(format_message(source_name, message))
 
+def format_outgoing_message(message):
+    usernames = re.search('@([^\s]+)', message)
```

Comment:
There is a problem here: this will replace all `@something` strings in a message, even if the part of the message is included in a verbatim ("`" or "```") section. We need to be smarter about that regular expression.